white-paper.md

```diff
@@ -48,16 +48,16 @@ We propose an immutable decentralized registry designed to distribute value base
 
-We predict that tea’s combination of tools, information, and rewards will justly incentivize developers, helping stimulate the growth of open-source software and fostering innovation.
+We predict that tea’s combination of tools, information, and rewards will fairly incentivize developers, helping stimulate the growth of open-source software and fostering innovation.
 
 [^4]: See: @w3
 
 ## The Decentralized Registry
 
-Every package manager has its own package registry duplicating the same metadata repeatedly. It’s time there was a single, comprehensive and definitive registry designed and governed by the communities that depend on it. This decentralized, immutable registry could provide security, stability and prevent
+Every package manager has its own package registry duplicating the same metadata repeatedly. It’s time there was a single, comprehensive, and definitive registry designed and governed by the communities that depend on it. This decentralized, immutable registry could provide security and stability and prevent
 malevolent intent.
 
-The Internet runs on tens of thousands of vital open-source components. It’s remarkable that thus far, incidents caused by the removal of essential open-source infrastructure have been minimal. The most famous was the removal of an NPM left-pad[^5] dependency in 2016, which cascaded into continuous integration and continuous deployment systems leaving developers high and dry for days. This event demonstrated that the Internet itself is based on fragile systems of development. Other examples involved active or intentional participation from the package maintainers sabotaging their popular packages (See colors.js, fakers.js[^6], and node-ipc[^7]), or bad actors looking to profit by pretending to help maintain packages and corrupting them to steal, for example, Bitcoin private keys (See event-stream[^8]).
+The Internet runs on tens of thousands of vital open-source components. It’s remarkable that thus far, incidents caused by the removal of essential open-source infrastructure have been minimal. The most famous was the removal of an NPM left-pad[^5] dependency in 2016, which cascaded into continuous integration and continuous deployment systems leaving developers unable to build their software for days. This event demonstrated that the Internet itself is based on fragile systems of development. Other examples involved active or intentional participation from package maintainers sabotaging their popular packages (See colors.js, faker.js[^6], and node-ipc[^7]), or bad actors looking to profit by pretending to help maintain packages and corrupting them to steal, for example, Bitcoin private keys (See EventStream[^8]).
```

Comment:
It wasn't clear what was meant by "high and dry". Now it's clear.

"fakers.js" => "faker.js"

The "event-stream" maintainers call the project "EventStream" in the documentation.

Reply:
The change to "unable to build their software" shows a more limited impact.
fakers.js vs. faker.js ==> definitely a typo. Thank you
event-stream vs. EventStream ==> That works. Was using the repo name, but we can use the project name just as well.

Reply:
> The change to "unable to build their software" shows a more limited impact.

This is a really good thing to highlight. I'm a native English speaker, and I know the connotation of the idiomatic expression, but I didn't know what was meant by "high and dry" in this context. It could be the case that leaving it open ended is the desired effect, and I'm happy to remove this change from the pull request, but it seemed like a good thing to scrutinize.

Reply:
I definitely appreciate the detailed review. Leaving it open-ended here is the desired effect.